addons/hr_holidays/tests/test_holidays_flow.py

```diff
@@ -43,13 +44,26 @@ def _check_holidays_status(holiday_status, ml, lt, rl, vrl):
             'categ_id': self.env['calendar.event.type'].sudo(self.user_hrmanager_id).create({'name': 'NotLimitedMeetingType'}).id
         })
         self.holidays_status_1 = HolidayStatusManagerGroup.create({
-            'name': 'NotLimited',
+            'name': 'NotLimitedHR',
```

Comment:
Those tests come from quite an old age :) . Shouldn't we have several unit tests for the various cases (various validation_type and user configuration), and an "integration-like" flow test that test a flow with one or two main configurations and flow ? It will help understanding the tests.

Reply:
After discussion note: we will launch a task to rewrite holidays tests after all holidays-related tasks are merged.